TrafficCapture/nettyWireLogging/src/main/java/org/opensearch/migrations/trafficcapture/netty/LoggingHttpRequestHandler.java

```diff
@@ -157,6 +171,7 @@ public void channelWritabilityChanged(ChannelHandlerContext ctx) throws Exceptio
     @Override
     public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws Exception {
         trafficOffloader.addExceptionCaughtEvent(Instant.now(), cause);
+        httpDecoderChannel.close();
```

Comment:
Do we need to close on normal termination as well?

Reply:
It shouldn't hurt, provided that we can identify what normal termination is - and I'm not 100% sure yet what exactly those cases are.  This close() is here not for resource management but because the connection and this handler should be in a terminal, non-recoverable state.  If we prematurely close the channel though and then have more data to write, that would be bad.  Given that the EmbeddedChannel doesn't hold any disposable resources, I'm fine letting the GC take the channel, pipeline, & handlers all out when this object gets finalized.  

I plan to address stress testing through many of the corner cases in the coming weeks, but it's hard to write representative tests without seeing some real traffic flow through the system first.